scripts/src/models/applications.ts

```diff
@@ -21,6 +21,13 @@ export type SignInType = "jwt" | "whitelist";
 export type ApplicationConfig = {
 	max_user_wallets: number | null;
 	sign_in_types: SignInType[];
+	limits: {
+		hourly_registration: number,
```

Comment:
types should end with a semicolon instead of a comma

Reply:
TS supports commas as types separator in object literals
[https://github.com/facebookarchive/esprima/pull/107](https://github.com/facebookarchive/esprima/pull/107)

Reply:
it supports them, but that's not our convention.
type definitions should end with semi-colons